COMMAND: JKQC 3sa2am2ma-1.log 3sa2am2ma-10.log 3sa2am2ma-11.log 3sa2am2ma-12.log 3sa2am2ma-13.log 3sa2am2ma-14.log 3sa2am2ma-15.log 3sa2am2ma-16.log 3sa2am2ma-17.log 3sa2am2ma-18.log 3sa2am2ma-19.log 3sa2am2ma-2.log 3sa2am2ma-20.log 3sa2am2ma-21.log 3sa2am2ma-22.log 3sa2am2ma-23.log 3sa2am2ma-24.log 3sa2am2ma-25.log 3sa2am2ma-26.log 3sa2am2ma-27.log 3sa2am2ma-28.log 3sa2am2ma-29.log 3sa2am2ma-3.log 3sa2am2ma-30.log 3sa2am2ma-31.log 3sa2am2ma-32.log 3sa2am2ma-33.log 3sa2am2ma-34.log 3sa2am2ma-35.log 3sa2am2ma-36.log 3sa2am2ma-37.log 3sa2am2ma-38.log 3sa2am2ma-39.log 3sa2am2ma-4.log 3sa2am2ma-40.log 3sa2am2ma-41.log 3sa2am2ma-42.log 3sa2am2ma-43.log 3sa2am2ma-44.log 3sa2am2ma-45.log 3sa2am2ma-46.log 3sa2am2ma-5.log 3sa2am2ma-6.log 3sa2am2ma-7.log 3sa2am2ma-8.log 3sa2am2ma-9.log -noexample -out database.pkl
COMMAND: JKQC -noexample -fc 100 -v 0.996 -sort gout -select 1 -out database.pkl
COMMAND: JKQC 3sa2am2ma-10.log 3sa2am2ma-11.log 3sa2am2ma-12.log 3sa2am2ma-13.log 3sa2am2ma-14.log 3sa2am2ma-15.log 3sa2am2ma-16.log 3sa2am2ma-17.log 3sa2am2ma-18.log 3sa2am2ma-19.log 3sa2am2ma-1.log 3sa2am2ma-20.log 3sa2am2ma-21.log 3sa2am2ma-22.log 3sa2am2ma-23.log 3sa2am2ma-24.log 3sa2am2ma-25.log 3sa2am2ma-26.log 3sa2am2ma-27.log 3sa2am2ma-28.log 3sa2am2ma-29.log 3sa2am2ma-2.log 3sa2am2ma-30.log 3sa2am2ma-31.log 3sa2am2ma-32.log 3sa2am2ma-33.log 3sa2am2ma-34.log 3sa2am2ma-35.log 3sa2am2ma-36.log 3sa2am2ma-37.log 3sa2am2ma-38.log 3sa2am2ma-39.log 3sa2am2ma-3.log 3sa2am2ma-40.log 3sa2am2ma-41.log 3sa2am2ma-42.log 3sa2am2ma-43.log 3sa2am2ma-44.log 3sa2am2ma-45.log 3sa2am2ma-46.log 3sa2am2ma-4.log 3sa2am2ma-5.log 3sa2am2ma-6.log 3sa2am2ma-7.log 3sa2am2ma-8.log 3sa2am2ma-9.log -noexample -out database.pkl
COMMAND: JKQC database.pkl -ct -gout
/home/kubeckaj/ACDB/database_v2/DLPNO_vnw16/Kubecka23-DFT/SYS_3sa_2am_2ma
slurmstepd: error: *** JOB 2640046 ON s95n83 CANCELLED AT 2023-03-08T13:48:21 ***
/home/kubeckaj/ACDB/database_v2/DLPNO_vnw16/Kubecka23-DFT/SYS_3sa_2am_2ma
COMMAND: JKQC 3sa2am2ma-10.log 3sa2am2ma-11.log 3sa2am2ma-12.log 3sa2am2ma-13.log 3sa2am2ma-14.log 3sa2am2ma-15.log 3sa2am2ma-16.log 3sa2am2ma-17.log 3sa2am2ma-18.log 3sa2am2ma-19.log 3sa2am2ma-1.log 3sa2am2ma-20.log 3sa2am2ma-21.log 3sa2am2ma-22.log 3sa2am2ma-23.log 3sa2am2ma-24.log 3sa2am2ma-25.log 3sa2am2ma-26.log 3sa2am2ma-27.log 3sa2am2ma-28.log 3sa2am2ma-29.log 3sa2am2ma-2.log 3sa2am2ma-30.log 3sa2am2ma-31.log 3sa2am2ma-32.log 3sa2am2ma-33.log 3sa2am2ma-34.log 3sa2am2ma-35.log 3sa2am2ma-36.log 3sa2am2ma-37.log 3sa2am2ma-38.log 3sa2am2ma-39.log 3sa2am2ma-3.log 3sa2am2ma-40.log 3sa2am2ma-41.log 3sa2am2ma-42.log 3sa2am2ma-43.log 3sa2am2ma-44.log 3sa2am2ma-45.log 3sa2am2ma-46.log 3sa2am2ma-4.log 3sa2am2ma-5.log 3sa2am2ma-6.log 3sa2am2ma-7.log 3sa2am2ma-8.log 3sa2am2ma-9.log -reacted -noexample -out database1DLPNO.pkl
Number of files: 46
COMMAND: JKQC database1DLPNO.pkl -ct -gout
/home/kubeckaj/ACDB/database_v2/DLPNO_vnw16/Kubecka23-DFT/SYS_3sa_2am_2ma
/home/kubeckaj/ACDB/database_v2/DLPNO_vnw16/Kubecka23-DFT/SYS_3sa_2am_2ma
COMMAND: JKQC 3sa2am2ma-10.log 3sa2am2ma-11.log 3sa2am2ma-12.log 3sa2am2ma-13.log 3sa2am2ma-14.log 3sa2am2ma-15.log 3sa2am2ma-16.log 3sa2am2ma-17.log 3sa2am2ma-18.log 3sa2am2ma-19.log 3sa2am2ma-1.log 3sa2am2ma-20.log 3sa2am2ma-21.log 3sa2am2ma-22.log 3sa2am2ma-23.log 3sa2am2ma-24.log 3sa2am2ma-25.log 3sa2am2ma-26.log 3sa2am2ma-27.log 3sa2am2ma-28.log 3sa2am2ma-29.log 3sa2am2ma-2.log 3sa2am2ma-30.log 3sa2am2ma-31.log 3sa2am2ma-32.log 3sa2am2ma-33.log 3sa2am2ma-34.log 3sa2am2ma-35.log 3sa2am2ma-36.log 3sa2am2ma-37.log 3sa2am2ma-38.log 3sa2am2ma-39.log 3sa2am2ma-3.log 3sa2am2ma-40.log 3sa2am2ma-41.log 3sa2am2ma-42.log 3sa2am2ma-43.log 3sa2am2ma-44.log 3sa2am2ma-45.log 3sa2am2ma-46.log 3sa2am2ma-4.log 3sa2am2ma-5.log 3sa2am2ma-6.log 3sa2am2ma-7.log 3sa2am2ma-8.log 3sa2am2ma-9.log -reacted -noexample -out database3DFT.pkl
Number of files: 46
/home/kubeckaj/ACDB/database_v2/DLPNO_vnw16/Kubecka23-DFT/SYS_3sa_2am_2ma
/home/kubeckaj/ACDB/database_v2/DLPNO_vnw16/Kubecka23-DFT/SYS_3sa_2am_2ma
COMMAND: JKQC 3sa2am2ma-10.log 3sa2am2ma-11.log 3sa2am2ma-12.log 3sa2am2ma-13.log 3sa2am2ma-14.log 3sa2am2ma-15.log 3sa2am2ma-16.log 3sa2am2ma-17.log 3sa2am2ma-18.log 3sa2am2ma-19.log 3sa2am2ma-1.log 3sa2am2ma-20.log 3sa2am2ma-21.log 3sa2am2ma-22.log 3sa2am2ma-23.log 3sa2am2ma-24.log 3sa2am2ma-25.log 3sa2am2ma-26.log 3sa2am2ma-27.log 3sa2am2ma-28.log 3sa2am2ma-29.log 3sa2am2ma-2.log 3sa2am2ma-30.log 3sa2am2ma-31.log 3sa2am2ma-32.log 3sa2am2ma-33.log 3sa2am2ma-34.log 3sa2am2ma-35.log 3sa2am2ma-36.log 3sa2am2ma-37.log 3sa2am2ma-38.log 3sa2am2ma-39.log 3sa2am2ma-3.log 3sa2am2ma-40.log 3sa2am2ma-41.log 3sa2am2ma-42.log 3sa2am2ma-43.log 3sa2am2ma-44.log 3sa2am2ma-45.log 3sa2am2ma-46.log 3sa2am2ma-4.log 3sa2am2ma-5.log 3sa2am2ma-6.log 3sa2am2ma-7.log 3sa2am2ma-8.log 3sa2am2ma-9.log -reacted -noexample -out database.pkl
Number of files: 46
/home/kubeckaj/ACDB/database_v2/DLPNO_vnw16/Kubecka23-DFT/SYS_3sa_2am_2ma
COMMAND: JKQC database.pkl -noexample -fc 100 -v 0.996 -sort g -select 3 -reacted -unique rg,el,g -out database3DFT.pkl
COMMAND: JKQC database.pkl -noexample -fc 100 -v 0.996 -globout -reacted -out database1DLPNO.pkl
COMMAND: JKQC database1DLPNO.pkl -ct -gout
/home/kubeckaj/ACDB/database_v2/DLPNO_vnw16/Kubecka23-DFT/SYS_3sa_2am_2ma
COMMAND: JKQC database.pkl -noexample -fc 100 -v 0.996 -sort gout -select 1 -reacted -out database1DLPNO.pkl
Traceback (most recent call last):
  File "/home/kubeckaj/Applications/JKCS2.1/JKQC/JKCS/lib/python3.9/site-packages/pandas/core/indexes/base.py", line 3361, in get_loc
    return self._engine.get_loc(casted_key)
  File "pandas/_libs/index.pyx", line 76, in pandas._libs.index.IndexEngine.get_loc
  File "pandas/_libs/index.pyx", line 108, in pandas._libs.index.IndexEngine.get_loc
  File "pandas/_libs/hashtable_class_helper.pxi", line 5198, in pandas._libs.hashtable.PyObjectHashTable.get_item
  File "pandas/_libs/hashtable_class_helper.pxi", line 5206, in pandas._libs.hashtable.PyObjectHashTable.get_item
KeyError: 'out'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/kubeckaj/Applications/JKCS2.1/JKQC/JKQCpickle.py", line 2042, in <module>
    sorted_indices = (clusters_df["log"]["gibbs_free_energy"]-clusters_df["log"]["electronic_energy"]+clusters_df["out"]["electronic_energy"]).sort_values().index
  File "/home/kubeckaj/Applications/JKCS2.1/JKQC/JKCS/lib/python3.9/site-packages/pandas/core/frame.py", line 3457, in __getitem__
    return self._getitem_multilevel(key)
  File "/home/kubeckaj/Applications/JKCS2.1/JKQC/JKCS/lib/python3.9/site-packages/pandas/core/frame.py", line 3508, in _getitem_multilevel
    loc = self.columns.get_loc(key)
  File "/home/kubeckaj/Applications/JKCS2.1/JKQC/JKCS/lib/python3.9/site-packages/pandas/core/indexes/multi.py", line 2922, in get_loc
    loc = self._get_level_indexer(key, level=0)
  File "/home/kubeckaj/Applications/JKCS2.1/JKQC/JKCS/lib/python3.9/site-packages/pandas/core/indexes/multi.py", line 3204, in _get_level_indexer
    idx = self._get_loc_single_level_index(level_index, key)
  File "/home/kubeckaj/Applications/JKCS2.1/JKQC/JKCS/lib/python3.9/site-packages/pandas/core/indexes/multi.py", line 2855, in _get_loc_single_level_index
    return level_index.get_loc(key)
  File "/home/kubeckaj/Applications/JKCS2.1/JKQC/JKCS/lib/python3.9/site-packages/pandas/core/indexes/base.py", line 3363, in get_loc
    raise KeyError(key) from err
KeyError: 'out'
COMMAND: JKQC database1DLPNO.pkl -ct -gout
/home/kubeckaj/ACDB/database_v2/DLPNO_vnw16/Kubecka23-DFT/SYS_3sa_2am_2ma
COMMAND: JKQC database.pkl -noexample -fc 100 -v 0.996 -sort gout -select 1 -reacted -out database1DLPNO.pkl
COMMAND: JKQC database1DLPNO.pkl -ct -gout
COMMAND: JKQC database1DLPNO.pkl -ct -gout -hout -s -fc 100 -v 0.996
COMMAND: JKQC database1DLPNO.pkl -movie
COMMAND: JKQC database1DLPNO.pkl -ct -glob -g -h -s -fc 100 -v 0.996
COMMAND: JKQC database3DFT.pkl -movie
COMMAND: JKQC database1DLPNO.pkl -ct -glob -g -h -s -fc 100 -v 0.996
COMMAND: JKQC database3DFT.pkl -movie
